src/ops/fixit.rs

```diff
@@ -162,12 +189,10 @@ fn run_rustfix(
             }
         }
 
-        let target = (target.clone(), package_id.clone());
-
-        if seen.contains(&target) {
+        if seen.contains(&build_unit) {
```

Comment:
Should this check be done before all of the others?  If w can get some of these things into `seen` then we can short-circuit once we've seen it